spec/lib/diaspora_federation/entities/comment_spec.rb

```diff
@@ -32,6 +32,8 @@ module DiasporaFederation
 
     it_behaves_like "a relayable Entity"
 
+    include_examples "#to_json output matches JSON schema"
```

Comment:
Ok, makes sense.

Or we can make it like `it_behaves_like "an XML Entity"`: `it_behaves_like "an JSON Entity"` ?